js/script.js

```diff
@@ -0,0 +1,35 @@
+$(function() {
+	var tweetLink = "https://twitter.com/intent/tweet?text=";
+	var quoteUrl = "https://quotesondesign.com/wp-json/posts?filter[orderby]=rand&filter[posts_per_page]=1";
+	var prefix = "https://cors-anywhere.herokuapp.com/";
+
+	function getQuote() {
+	    $.getJSON(prefix + quoteUrl, createTweet);
+	    $.ajaxSetup({ cache: false });
+	}
+
+	function createTweet(input) {
+	    var data = input[0];
```

Comment:
before you assign first element of your response, you should check that response is an array and have at least one element